misc/config_tools/scenario_config/pt_intx_c.py

```diff
@@ -17,7 +17,7 @@ def generate_file(vm_info, config):
 
     print("{}".format(scenario_cfg_lib.HEADER_LICENSE), file=config)
     print("", file=config)
-    print("#include <x86/vm_config.h>", file=config)
+    print("#include <asm/vm_config.h>", file=config)
```

Comment:
Do you regenerate the files under  misc\config_tools\data\generic_board\generic_code\<scenario_name> to reflect your changes of config_tools?